docs/docusaurus/multitenancy/admin-authz-ext-iam.md

```diff
@@ -0,0 +1,142 @@
+# Admin Authorization with External IAM
+
+In the past tutorials, the administrator utilized a static API key for authentication and authorization.
+The Agent also features a more sophisticated role-based authorization mechanism using external IAM,
+allowing users to be authorized under different roles and granting access to different parts of the Agent.
+
+## Roles
+
+For the Agent admin authorization, we need to distinguish the administrator of each component
+
+1. [Agent Administrator](/docs/concepts/glossary#administrator)
+2. Keycloak Administrator
+
+These roles may also be represented by the same person.
+
+## Prerequisites
+
+1. Keycloak up and running
+2. Keycloak is configured as follows
+   1. A realm called `my-realm` is created
+   2. A client called `prism-agent` under `my-realm` with __authorization__ feature is created. (See [create client instruction](https://www.keycloak.org/docs/latest/authorization_services/index.html#_resource_server_create_client))
+   3. Make sure the `prism-agent` client has __direct access grants__ enabled to simplify the login
+3. PRISM Agent up and running
+4. PRISM Agent is configured with the following environment variables:
+   1. `KEYCLOAK_ENABLED=true`
+   2. `KEYCLOAK_URL=http://localhost:9980` (replace with appropriate value)
+   3. `KEYCLOAK_REALM=my-realm`
+   4. `KEYCLOAK_CLIENT_ID=prism-agent`
+   5. `KEYCLOAK_CLIENT_SECRET=<KEYCLOAK_CLIENT_SECRET>` (replace with appropriate value)
+   6. `KEYCLOAL_ROLES_CLAIM_PATH=resource_access.prism-agent.roles`
```

Comment:
I wonder if it makes sense to create a directory, `tutorials` in the OEA directory with the scripts and docker-compose files that can be used to go through the tutorial.
I think it might simplify the life for users who want to test a particular capability

Reply:
Not now; I think we can propose this improvement at AoH or Engineering meeting.
as for me, it would really help if we add `tutorials` and `demo` directories with the instructions and workable setup for the useres.